packages/apkeditor/build.sh

```diff
@@ -0,0 +1,31 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/REAndroid/APKEditor
+TERMUX_PKG_DESCRIPTION="Powerful Android APK editor - aapt/aapt2 independent."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="@Veha0001"
+TERMUX_PKG_VERSION=1.4.2
+TERMUX_PKG_SRCURL=https://github.com/REAndroid/APKEditor/releases/download/V${TERMUX_PKG_VERSION}/APKEditor-${TERMUX_PKG_VERSION}.jar
```

Comment:
The appears to be a precompiled release JAR.
That is *fine* for Java I suppose, but it's unusually since we usually compile packages from source.